pom.xml

```diff
@@ -340,6 +340,8 @@ under the License.
             <exclude>**/docs/**</exclude>
             <!-- dag files -->
             <exclude>**/dag/**</exclude>
+            <exclude>**/MANIFEST.MF</exclude>
+            <exclude>**/*.proto</exclude>
```

Comment:
What's these files?

Reply:
In some environments, I've experienced a number of protobuf files being generated, and had become a trouble while building the project.